src/api/services/system/dotcms-config.ts

```diff
@@ -93,4 +93,13 @@ export class DotcmsConfig extends CoreWebService {
     getNavigationMenu(): Array<any> {
         return this.configParams.menu;
     }
+
+    /**
+     * Returns the default rest page count to display.
+     *
+     * @returns <number> The max number of sites to display after a search.
+     */
+    getDefaultRestPageCount(): number {
+        return this.configParams.config['DEFAULT_REST_PAGE_COUNT'];
```

Comment:
shouldn't be a constant

const DEFAULT_REST_PAGE_COUNT = 'DEFAULT_REST_PAGE_COUNT';